plugins/package-managers/bundler/src/main/kotlin/Bundler.kt

```diff
@@ -358,8 +358,11 @@ class Bundler(
         workingDir.walk().maxDepth(1).filter { it.isFile && it.extension == "gemspec" }.firstOrNull()
 
     private fun queryRubygems(name: String, version: String, retryCount: Int = 3): GemSpec? {
-        // See http://guides.rubygems.org/rubygems-org-api-v2/.
-        val url = "https://rubygems.org/api/v2/rubygems/$name/versions/$version.yaml"
+        // NOTE: We explicitly use platform=ruby here to enforce the same behavior here that we also have in our
+        // Bundler resolving logic.
+        // See <https://github.com/oss-review-toolkit/ort/blob/main/plugins/package-managers/bundler/src/main/resources/resolve_dependencies.rb#L31-L32>
+        // See <http://guides.rubygems.org/rubygems-org-api-v2/>.
+        val url = "https://rubygems.org/api/v2/rubygems/$name/versions/$version.yaml?platform=ruby"
 
         return okHttpClient.downloadText(url).mapCatching {
```

Comment:
Merge commits are not allowed in this repository. Please do rebase this commit onto head of main instead.